Simplified/Book/UI/NYPLBookDetailView.m

```diff
@@ -594,6 +594,11 @@ - (void)setState:(NYPLBookState)state
       self.buttonsView.hidden = NO;
       self.normalView.state = NYPLBookButtonsStateDownloadSuccessful;
       self.buttonsView.state = NYPLBookButtonsStateDownloadSuccessful;
+      if (self.downloadStarted && [self.window isKeyWindow]) {
+        // The "isKeyWindow" check avoids focusing on book detail view if it is not visible
+        [self.buttonsView setReadButtonAccessibiltyLabelWithMessage:self.normalView.messageStringForNYPLBookButtonStateSuccessful];
+        [self setUIAccessibilityFocusToView:self.buttonsView];
```

Comment:
this will trigger reading our custom message, correct?

Reply:
Right